### DISCUSSION TITLE
Anyone have a åskväder?

### DISCUSSION THREAD

**Discussion by honglihu1986** [basilfx/TRADFRI-Hacking#59] [Q&A]
![Screenshot_20230523-181451_Gallery](https://github.com/basilfx/TRADFRI-Hacking/assets/39935925/0cd25389-adc5-4d8f-8679-93962fff76ba)

**Comment by PerMejdal** [comment]
I have one, but I don't intent to destroy it.

**Comment by honglihu1986** [comment]
I can not find the OTA file.

**Comment by MattWestb** [comment]
TF is having it in its feed but i dont knowing if its working then i have not testing it on my device.
<html>
<body>


10078247 | 0x110D 4365 | ASKVADER on/off switch (E1836) | 3 ZB3 | 1.0.002 1.0.32 ✅ | 🔲
-- | -- | -- | -- | -- | --
